content/getting-started/setup.textile

```diff
@@ -192,19 +192,12 @@ Run the following to instantiate a client:
 ```[realtime_javascript]
 // Using promises
 const realtime = new Ably.Realtime.Promise({ key: apiKey });
-
-// Using callbacks
-const realtime = new Ably.Realtime({ key: apiKey });
 ```
 
 ```[realtime_nodejs]
 // Using promises
-const Ably = require('ably');
+const Ably = require('ably/promises');
```

Comment:
Don't need this if we're using the `Realtime.Promise` constructor
```suggestion
const Ably = require('ably');
```

Reply:
https://github.com/ably/docs/pull/2119/commits/78c9d6195c3137620f38a05e13839a01aa68fd5c
I also realised some of the other setup JS code samples didn't include the require. So updated those at the same time.